cps_stage4/extrapolation.py

```diff
@@ -78,8 +83,8 @@ def increment_year(self, tol=0.01):
             setattr(self, '{}_benefits'.format(benefit), benefits)
 
     @staticmethod
-    def _extrapolate(WT, I, benefits, prob, target,
-                     benefit_name, benefit_year, tol=0.01, J=15):
+    def _extrapolate(WT, III, benefits, prob, target,
+                     benefit_name, benefit_year, tol, JJJ=15):
```

Comment:
@martinholmer why did you change the variable names here?

Reply:
To avoid the pucodestyle warnings about ambiguous variable name.
See [this list](https://github.com/open-source-economics/taxdata/issues/232#issuecomment-399711170) of warnings.

Reply:
I see, but `III` doesn't seem to add any more information about what the matrix is than the information given by `I`. Could we use something like `indicator` or `ind`, given that this is an indicator matrix. `JJJ` could be `num_members_in_unit` or something else like that. Does that seem sensible?

Reply:
@hdoupe said:
> I see, but `III` doesn't seem to add any more information about what the matrix is than the information given by `I`. 

Yes, you are absolutely right about that.

> Could we use something like `indicator` or `ind`, given that this is an indicator matrix. `JJJ` could be `num_members_in_unit` or something else like that. Does that seem sensible?

I like your proposed rename of `JJJ` because the variable name says what it is.
But changing `III` to `indicator` still leaves readers of the script wondering what is being indicated.
What does `III` indicate?

Reply:
Ah, right. By "indicator" I mean: indicator of whether the person is participating or not. How about `participating_switch` or `participating_indicator`? These are still pretty long though.

Reply:
@hdoupe said:
> `JJJ` could be `num_members_in_unit` or something else like that.
Do you mean number of people in a filing unit?  If so, that doesn't seem right given the code.
For example, what does `JJJ=15` mean?  Can you clarify?

Reply:
My understanding is that `J=15` means that there are at a maximum 15 members in each unit. That means that the maximum width of the participation indicator matrix is 15.